configure.ac

```diff
@@ -3,20 +3,20 @@
 # The Universal Permissive License (UPL), Version 1.0
 #
 # Subject to the condition set forth below, permission is hereby granted to any person obtaining a copy of this software,
-# associated documentation and/or data (collectively the "Software"), free of charge and under any and all copyright rights in the 
```

Comment:
There are differences because of trailing white spaces. My vi is configured to remove them before every save (so I do not have to worry about that). So it is my fault, sorry. I will bring those back.